schema.sql

```diff
@@ -1,5 +1,15 @@
-/* Database schema to keep the structure of entire database. */
+-- Database schema to keep the structure of entire database.
 
-CREATE TABLE animals (
-    name varchar(100)
+CREATE DATABASE vet_clinic;
+CREATE TABLE animals(
+    id INT,
```

Comment:
[OPTIONAL] I suggest you make this column your primary key. Having primary keys will assist you link to other table by using your primary keys as foreign key of other tables.